hermes-consumers/build.gradle

```diff
@@ -1,13 +1,14 @@
 configurations {
     sbeClasspath
 }
-
+//    TODO: use implementation instead of compile
 dependencies {
     compile project(':hermes-common')
     compile project(':hermes-tracker')
     compile project(':hermes-metrics')
     compile project(':hermes-schema')
 
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter', version: '2.4.2'
```

Comment:
It seems like `hermes-management` is using more than only `spring-boot-starter` dependency, so it would not do us any good, as far as I understand it.

Reply:
By the version I meant: `2.4.2`, which is already defined here: https://github.com/allegro/hermes/blob/master/hermes-management/build.gradle#L1